README.md

```diff
@@ -171,24 +172,26 @@ With Thread, you can change the operating mode of the client from the default ro
 
 #### Supported boards
 
-* UBLOX_EVK_ODIN_W2. Check instructions for compilation [here](#compile-configuration-for-odin-wifi).
+* UBLOX_EVK_ODIN_W2. Check instructions for compilation [here](#compile-configuration-for-odin-and-realtek-wifi).
```

Comment:
Read the [instructions for compilation](#compile-configuration-for-odin-and-realtek-wifi). And the same for the other Realtek case.